torchgeo/datasets/geo.py

```diff
@@ -75,6 +75,13 @@ class GeoDataset(Dataset[dict[str, Any]], abc.ABC):
     _crs = CRS.from_epsg(4326)
     _res = 0.0
 
+    #: Glob expression used to search for files.
+    #:
+    #: This expression should be specific enough that it will not pick up files from
+    #: other datasets. It should not include a file extension, as the dataset may be in
+    #: a different file format than what it was originally downloaded as.
+    filename_glob = "*"
```

Comment:
This now need to be defined here in GeoDataset although it is only used in Raster and Vector

Reply:
Yes, that's fine.